default.build

```diff
@@ -36,11 +36,11 @@
     <call target="build_windowsphone81" />
     <call target="build_windows10" />
     <call target="build_psm" />
-    <call target="build_web" />
```

Comment:
What was failing about this?  This builds currently on the build server, so any error here is with your changes in this PR or a local problem on your machine.